lib/jsonwp-proxy/proxy.js

```diff
@@ -26,15 +26,15 @@ class JWProxy {
   }
 
   endpointRequiresSessionId (endpoint) {
-    return !_.includes(['/session', '/sessions', '/status'], endpoint);
+    return !_.includes(['/session', '/sessions', '/status', '/homescreen'], endpoint);
```

Comment:
It can be proxied to WDA without a session. I'm just curious as to why, in Appium, it should be possible to do it.

It will require significant energy to get it to work in Appium, since Appium does not have a connection to the device without a session. All the current sessionless commands do nothing to the device.